### PR TITLE
Add classic support

### PR DESCRIPTION
**Pull request by JulieSFL**

### CODE REVIEW

File: standard_plus_issue/data/template_email_standard_plus_issue.xml [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+        <h3>Request ${object.name}</h3>
+        <p>
+          A request of type <strong>${object.type}</strong> for project ${object.user_id.company_id.name} has been submitted by ${(object.user_id.name)} with a level '${object.priority}' priority.
+        </p>
+        <p>
+          <strong>Concerned module:</strong>
+          % if object.module_id.name:
+            ${object.module_id.name}
+          % else:
+            left blank
```
Comment (ghost) [inline standard_plus_issue/data/template_email_standard_plus_issue.xml:21]: what does `left blank` do ?

File: standard_plus_issue/models/standard_plus_issue.py [hunk 2]
```diff
@@ -0,0 +1,188 @@
...
+    state = fields.Selection(
+        string='State',
+        selection=[
+            ('draft', 'Draft'),
+            ('escalated', 'Escalated'),
+            ('submitted', 'Submitted'),
+            ('addressed', 'Addressed'),
+            ('rejected', 'Rejected'),
+        ],
+        default='draft',
```
Comment (ghost) [inline standard_plus_issue/models/standard_plus_issue.py:31]: required=True ?

**Review by ghost — COMMENTED** [review]
@JulieSFL I understand you will submit the translations later ? I wont block for translations.

Though, it is important to implement security for this module. What group can click on what ? Will the changes to a ticket be tracked ? Can anyone edit any ticket at any time ?

File: standard_plus_issue/data/template_email_standard_plus_issue.xml (hunk 1 above)
Comment (JulieSFL) [reply standard_plus_issue/data/template_email_standard_plus_issue.xml:21]: It literally prints left blank. (eg: Concerned module: left blank). It indicates that the user didn't fill the field.

**Comment by JulieSFL** [comment]
@dufresnedavid Groups are for this task: https://projects.savoirfairelinux.com/issues/105572
Translations are here

File: standard_plus_issue/models/standard_plus_issue.py (hunk 2 above)
Comment (JulieSFL) [reply standard_plus_issue/models/standard_plus_issue.py:31]: Can you explain why you think it's needed?

**Comment by JulieSFL** [comment]
Inherited module standard_plus_issue_studio [here](https://github.com/JulieSFL/standard-plus-addons/tree/julie_studio_issue_105539/standard_plus_issue_studio)

File: standard_plus_issue/models/standard_plus_issue.py (hunk 2 above)
Comment (ghost) [reply standard_plus_issue/models/standard_plus_issue.py:31]: to prevent a issue from having no state

**Review by ghost — APPROVED** [review]

**Comment by ghost** [comment]
:+1:

File: standard_plus_issue/models/standard_plus_issue.py [hunk 3]
```diff
@@ -0,0 +1,203 @@
...
+    _name = 'standard.plus.issue'
+    _description = __doc__
+    _inherit = ['mail.thread']
+
+    name = fields.Char(
+        string='Reference',
+        readonly=True,
+    )
+
+    state = fields.Selection(
```
Comment (joaoalf) [inline standard_plus_issue/models/standard_plus_issue.py:22]: @JulieSFL State fields should also be readonly.

**Comment by joaoalf** [comment]
@JulieSFL I prefer that you remove the commit 3c5c78af04bf8bff806023ae4056e0f24389419c. I will add the functionality in another PR. Instead, can you please add a menu with a listview and form view in Settings -> Technical

**Review by joaoalf — APPROVED** [review]
:+1:

**Pull request merged**